angularjs-portal-frame/src/main/webapp/js/app-config.js

```diff
@@ -22,9 +23,11 @@ define(['angular'], function(angular) {
             'isWeb' : false,
             'searchURL' : '/web/apps/search/'
         })
-        ;
+        .constant('NOTIFICATION', {
+            'groupFiltering' : true,
```

Comment:
Interesting that whether it filters by groups is configurable.  Would one ever not filter by groups?  In a future when the notifications URL does filtering server-side so that there's no further filtering to do client-side?